lib/screens/create_region_screens/viewmodels/create_region_bloc.dart

```diff
@@ -14,9 +17,26 @@ class CreateRegionBloc extends Bloc<CreateRegionEvent, CreateRegionState> {
     on<OnNextTapped>(_onNextTapped);
     on<OnCreateRegionTapped>(_onCreateRegionTapped);
     on<OnBackPressed>(_onBackPressed);
+    on<UploadImage>(_uploadImage);
     on<ClearCreateRegionPageCommand>((_, emit) => emit(state.copyWith()));
   }
 
+  Future<void> _uploadImage(UploadImage event, Emitter<CreateRegionState> emit) async {
```

Comment:
is this method really "uploading an image?"

Reply:
Maybe `OnImagePicked`